src/main/webapp/app/pages/companyPage/CompanyPage.tsx

```diff
@@ -516,13 +516,12 @@ export default class CompanyPage extends React.Component<ICompanyPage> {
                       <Col>
                         <div>Quick Tools</div>
                         <div>
-                          <QuickToolButton>
-                            <Link
-                              to={`/companies/${this.company.id}/create-users`}
-                            >
-                              Create Company Users
-                            </Link>
-                          </QuickToolButton>
+                          <Link
+                            className="btn btn-outline-primary"
```

Comment:
This button is missing the `m-2` that is used in QuickToolButton. I think we should probably still use that component. To link to another page, you can use the `routing` prop to change the history. See the UserPage.tsx file for an example.

Reply:
I think the missing margin is also why the Screenshot Test is also failing.

Reply:
I tried using the history prop, but it's breaking the back button.
You not have to hit it twice to get the browser to go back.
Are you sure we need it?

Reply:
Oh that's interesting. Have you tried just using the `QuickButtonTool` with an onClick. Maybe adding the history prop call with the `Link` component is causing some issues?

Reply:
If that doesn't work, I think the solution you have now is good too.

Reply:
What's it for? Shouldn't the like handle the routing history?

Reply:
I imagine the QuickToolButton was created so the styling can be reused for all quick tools.

The Link can be used to handle redirecting. I believe the implementation of that component probably pushes to the history to change the location. We can also update the history by directly calling `history.push()`.

I don't see a problem with either approach:
a. Copy quick tool button styles to Link component
b. Or using history to redirect in QuickToolButton component